run-build.ps1

```diff
@@ -21,6 +22,7 @@ if($Help)
     Write-Host "  -Configuration <CONFIGURATION>     Build the specified Configuration (Debug or Release, default: Debug)"
     Write-Host "  -Architecture <ARCHITECTURE>       Build the specified architecture (x64 or x86 (supported only on Windows), default: x64)"
     Write-Host "  -NoPackage                         Skip packaging targets"
+    Write-Host "  -NoBuild                           Perform repo initialization without building the product"
```

Comment:
What does "repo initialization" means?, It is no clear that I'm calling run-build, but I'm asking not to build with  `nobuild`.